### ISSUE TITLE
Some Skills doesn't seems to work.

### ISSUE THREAD

**Issue by dinuovo** [BeetoGuy/LevelUp2#110]
Hello i don't know if the mod author is still active i hope it is, i have a problem with some skills; Cooking Speed, Smelting efficiency, Crafting specialization, Sword damage, Combat experience as far as i know are not working. I binded the furnace, i tried doing some experiment with and without the smelting skill at max level, the speed doesn't change. I don't get any experience from crafting. Sword damage always seems the same, the only thing that is working is sprint speed. I tried to go on the config file but i can't see the problem, the skills are all enabled and active. Is there a config file to tune the actual percentage of the various skills? I didn't try other skills yet, i'm currently playing on the last version of MC eternal. Please help.

**Comment by Draconox99** [comment]
Hi, i have exactly the same things to report, 1.4.4 Mc Eternal version for me, but with the last version levelup 1.5.8

**Comment by dinuovo** [comment]
> Hi, i have exactly the same things to report, 1.4.4 Mc Eternal version for me, but with the last version levelup 1.5.8

I also have the same levelup version.

**Comment by dinuovo** [comment]
> Hi, i have exactly the same things to report, 1.4.4 Mc Eternal version for me, but with the last version levelup 1.5.8

I found the solution, at least for my case.
If you go in the control settings and look for the level up section there is a key that's called enable level up skills or something like that, in my case I had to change the keyboard button because it was in conflict, but then in game when I pressed it the furnaces, sword damage and all the skill started to work, it even started to drop ore chunk from mining and it was new for me. If this method  work I suggest you to decrease the level of stonecutting skill because it's broken, with a click i deleted almost an entire chunk with my aoe pickaxe and it wasn't even at the max rank. Hope it works for you to, have a good day.
